gallery/src/components/NxTable/NxTablePage.tsx

```diff
@@ -243,6 +245,13 @@ export default function NxTablePage() {
         An example with a sortable column.
       </GalleryExampleTile>
 
+      <GalleryExampleTile title="Filter Columns Example"
+                          id="nx-table-filter-example"
+                          liveExample={NxTableFilterExample}
+                          codeExamples={tableFilterExample}>
+        An example with filter columns.
```

Comment:
I'd elaborate on this a little bit, mentioning that the first column's filter demonstrates a basic, free-form filter input while the second column's filter demonstrates a filter input with a datalist for suggestion/dropdown capability..

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/8ea829fdaa8eec82f98998a15d950c3282c30f9f

Reply:
Fixed: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/44e2eb852a34641822edbe5670ddf0439f11771c